com.yacreader.YACReader.yml

```diff
@@ -0,0 +1,123 @@
+          - touch pdfium/build/config/gclient_args.gni
+          # Exclude test fonts from build
+          - sed -i '/"\/\/third_party\/test_fonts",/s/^/#/g' pdfium/testing/BUILD.gn
+          # Build
+          - gn --root=pdfium gen pdfium/out/Release --args="is_clang=false use_sysroot=false is_debug=false symbol_level=0 pdf_enable_v8=false pdf_enable_xfa=false treat_warnings_as_errors=false use_system_libjpeg=true use_system_zlib=true pdf_bundle_freetype=false use_system_freetype=true use_system_lcms2=true use_system_libpng=true use_custom_libcxx=false pdf_is_standalone=true use_system_libopenjpeg2=true is_component_build=true use_gold=false"
+          - ninja -C pdfium/out/Release pdfium
+          - sed -e 's/@VERSION@/5005/g' -e 's#^\s*prefix\s*=.*$#prefix='"$FLATPAK_DEST"'#' -i libpdfium.pc
+          # Install
+          - install -D -m644 pdfium/LICENSE "$FLATPAK_DEST/share/licenses/pdfium/LICENSE"
+          - install -D pdfium/public/*.h --target-directory="$FLATPAK_DEST/include/pdfium"
+          - install -D pdfium/public/cpp/* --target-directory="$FLATPAK_DEST/include/pdfium/cpp"
+          - install -D pdfium/docs/* --target-directory="$FLATPAK_DEST/share/doc/pdfium"
+          - install -Dm755 pdfium/out/Release/libpdfium.so --target-directory="$FLATPAK_DEST/lib"
+          - install -Dm644 libpdfium.pc --target-directory="$FLATPAK_DEST/lib/pkgconfig"
+        sources:
+          - type: git
+            url: https://pdfium.googlesource.com/pdfium
+            # curl 'https://omahaproxy.appspot.com/linux?channel=stable' | cut -d'.' -f 3
+            branch: chromium/5195
+            dest: pdfium
+          - type: git
+            url: https://chromium.googlesource.com/chromium/src/build.git
+            # awk '/build_revision/ {print substr($2,2,40)}' pdfium/DEPS
+            commit: 26f8da34750ac3bccf683ed5f70d86f21c54b22b
+            dest: pdfium/build
+          - type: git
+            url: https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp
+            # awk '/abseil_revision/ {print substr($2,2,40)}' pdfium/DEPS
+            commit: 62a4d6866aeeca02036c510b2f3f286084dd62af
+            dest: pdfium/third_party/abseil-cpp
+          - type: file
+            url: https://chromium.googlesource.com/chromium/src/+/master/tools/generate_shim_headers/generate_shim_headers.py?format=TEXT
+            dest-filename: generate_shim_headers.py
+            sha256: f95f0d29e3e12b22a7b86455265c87bc0219a25c09a7fa160d34b608992ea6da
+          - type: file
+            url: https://aur.archlinux.org/cgit/aur.git/plain/libpdfium.pc?h=libpdfium-nojs
+            dest-filename: libpdfium.pc
+            sha256: 4e8cf32c09568ae4c59244f0a221eee8f87ea4a31ddb31f796a1d818c967d477
+        modules:
+          - name: gn
+            buildsystem: simple
+            build-options:
+              env:
+                CC: gcc
+                CXX: g++
```

Comment:
do you really need to set this?

Reply:
Maybe. Depends. The gn bootstrap build system uses environment variables to override default settings. The default in this case is clang. Setting it to gcc saves a build dependency.

Reply:
there is a llvm SDK extension if it really want llvm/clang.